spring-integration-file/src/main/java/org/springframework/integration/file/tail/FileTailingMessageProducerSupport.java

```diff
@@ -96,6 +108,11 @@ protected TaskExecutor getTaskExecutor() {
 	}
 
 	@Override
+	public void setTaskScheduler(TaskScheduler taskScheduler) {
```

Comment:
i thought that also , but i saw it in aggregation so i though i should add it. i will remove it